google-cloud-bigtable/pom.xml

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>google-cloud-bigtable</artifactId>
+  <version>0.26.1-beta-SNAPSHOT</version>
+  <packaging>jar</packaging>
+  <name>Google Cloud Bigtable</name>
+  <url>https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-bigtable</url>
+  <description>
+    Java idiomatic client for Google Cloud Bigtable.
+  </description>
+  <parent>
+    <groupId>com.google.cloud</groupId>
+    <artifactId>google-cloud-pom</artifactId>
+    <version>0.26.1-alpha-SNAPSHOT</version>
+  </parent>
+  <properties>
+    <site.installationModule>google-cloud-bigtable</site.installationModule>
+    <artifact.version>${project.version}</artifact.version>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-tcnative-boringssl-static</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>google-cloud-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>google-cloud-core-grpc</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>proto-google-cloud-bigtable-v2</artifactId>
```

Comment:
Why don't you need proto-google-cloud-bigtable-admin-v2 also?

Reply:
Correct, and also added grpc bigtable-admin dependency.